server/vid.py

```diff
@@ -0,0 +1,62 @@
+import bentoml
+from bentoml.io import Image, JSON
+from fastapi import FastAPI, Form, UploadFile, File
+from fastapi.responses import FileResponse
+import uvicorn
+from typing_extensions import Annotated
+from diffusers.utils import load_image, export_to_video
+import torch
+from io import BytesIO 
+import PIL
+import math
+
+MODEL_ID= "stabilityai/stable-video-diffusion-img2vid-xt"
+
+bentoml.diffusers.import_model(
+        "svd-xt",
+        MODEL_ID
+)
+
+bento_model = bentoml.diffusers.get("svd-xt:latest")
+svd_runner = bento_model.to_runner()
+svd_runner.init_local()
+
+svc = bentoml.Service("svd-xt",
+        runners=[svd_runner])
+
+@svc.api(input=Image(), output=Image())
+def img2vid(input_data):
+    res = svd_runner.run(**input_data)
+    return res[0][0]
+
+app = FastAPI(root_path="/v9")
+svc.mount_asgi_app(app)
+
+#@app.get("/status")
```

Comment:
Include status endpoint for observability
```suggestion
@app.get("/status")
```